scripts/check_health.py

```
@@ -0,0 +1,54 @@
+import sys
+from pathlib import Path
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="Perform basic CyberPot health checks")
+    parser.add_argument("--compose-file", default="docker-compose.yml", help="Compose file to inspect")
+    return parser.parse_args()
+
+
+def run_command(command: list[str], env: dict[str, str] | None = None) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(command, capture_output=True, text=True, check=False, env=env)  # nosec B603 - command list is not shell-interpreted
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)